tldw_Server_API/app/api/v1/endpoints/flashcards.py

```
@@ -2039,7 +2039,7 @@ def export_flashcards(
     include_workspace_items: bool = False,
     tag: Optional[str] = None,
     q: Optional[str] = None,
-    format: Optional[str] = Query("csv", pattern="^(csv|apkg)$"),
+    format: Optional[str] = Query("csv", pattern="^(csv|apkg|json)$"),
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)